modules/product/config/optional/core.entity_form_display.commerce_line_item.product_variation.default.yml

```diff
@@ -22,14 +22,10 @@ content:
     type: number
     weight: 1
     settings:
-      placeholder: ''
```

Comment:
We should not be removing the unit price here, that means you can't use it in the IEF on the order add/edit page.